content/articles/adding-custom-methods-for-linq-queries/index.md

```diff
@@ -0,0 +1,161 @@
+### Adding custom methods for LINQ queries (C#) 
+### Introduction
+By introducing extension methods to the IEnumerableT> interface, you may expand the set of methods available for LINQ queries. Any data sequence can be converted into one single value by creating your custom aggregate method in SQL Server. 
+
+You can also build a method that returns a new series of values and acts as a custom filter or a specialized data transform for a sequence of variables. 
+
+A few examples include the Distinct and Skip techniques, as well as the Reverse.
+
+You may apply your methods to any enumerable collection when you extend the IEnumerableT> interface. In this article, we shall deeply take a look at various custom methods that are used in LINQ queries in C#.

```

Comment:
change "In this article, we shall deeply take a look at various custom methods that are used in LINQ queries in C#." to "In this article, we shall look at various custom methods used in LINQ queries in C#."